gpjax/likelihoods.py

```diff
@@ -199,6 +199,44 @@ def predict(self, dist: tfd.Distribution) -> tfd.Distribution:
         return self.link_function(mean / jnp.sqrt(1.0 + variance))
 
 
+@dataclass
+class Beta(AbstractLikelihood):
+    def link_function(self, f: Float[Array, "..."]) -> tfd.Distribution:
+        r"""The probit link function of the Beta likelihood.
+
+        Args:
+            f (Float[Array, "..."]): Function values.
+
+        Returns
+        -------
+            tfd.Distribution: The likelihood function.
+        """
+
+        return tfd.Beta(inv_probit(f[0]), inv_probit(f[1]))
+
+    def predict(self, dist: tfd.Distribution) -> tfd.Distribution:
+        r"""Evaluate the pointwise predictive distribution.
+
+        Evaluate the pointwise predictive distribution, given a Gaussian
+        process posterior and likelihood parameters.
+
+        Args:
+            dist (tfd.Distribution): The Gaussian process posterior, evaluated
+                at a finite set of test points.
+
+        Returns
+        -------
+            tfd.Distribution: The pointwise predictive distribution.
+        """
+        mean = dist.mean().ravel()
+        stddev = dist.stddev().ravel()
+
+        alpha = mean / stddev
+        beta = (1.0 - mean) / stddev
+
+        return self.link_function([alpha, beta])
```

Comment:
I don't believe the predictive marginal $p(y*) = \int p(y*| f*) p(f* | Data)$  can be done in closed form here the beta likelihood. (Or can it @thomaspinder?) If there is no analytical expression, you will need to either draw samples from the posterior distribution and use Monte, our use our quadrature integrator on the link functions's `log_prob` function $p(y_i|f_i)$.